crates/typst-library/src/layout/columns.rs

```diff
@@ -5,27 +5,21 @@ use crate::engine::Engine;
 use crate::foundations::{elem, Content, NativeElement, Packed, Show, StyleChain};
 use crate::layout::{BlockElem, Length, Ratio, Rel};
 
-/// Separates a region into multiple equally sized columns.
+/// 複数の同じ大きさを持つ段への領域の分割。
```

Comment:
修飾の順番を変えると読みやすいかもしれません。
```suggestion
/// 同じ大きさを持つ複数の段に領域を分割。
```

Reply:
ありがとうございます。
いろいろ悩んだのですが「領域を等幅の複数段に分割。」としてみました。

Reply:
ご検討ありがとうございました。resolveとさせていただきます。